### ISSUE TITLE
PHP Notice:  Undefined variable: openTime

### ISSUE THREAD

**Issue by syndrael** [jaggedsoft/php-binance-api#153]
**PHP Notice:  Undefined variable: openTime**
- bug: When $ticks is an empty array, a PHP notice is displayed
- feature: function candlesticks()
- enhancement: add this to existing features

**Short Description:**
PHP Notice:  Undefined variable: openTime in /[..]/vendor/jaggedsoft/php-binance-api/php-binance-api.php on line 1237
  
**Platform:**
- linux

**php version:**
- 7.0

**code**
```php
foreach ($ticks as $tick) {
            list(**$openTime**, [..]) = $tick;
            $output[$openTime] = [
             (..)
            ];
        }
        $this->info[$symbol][$interval]['firstOpen'] = **$openTime**;
```

Regards from Paris, France

**Comment by dmzoneill** [comment]
Hi can you please update your software.  You are on an old version
Many thanks

https://github.com/jaggedsoft/php-binance-api/commit/7033c50ca5a17c76396cd061412e5475a36de7f0

**Comment by dmzoneill** [comment]
@jaggedsoft  can you push a release? for composer

**Comment by jaggedsoft** [comment]
Available in `v0.4.2`

Many improvements and bug fixes: 16 commits by dmzoneill 🏆
Thank you for your hard work mate
